### PR TITLE
[Community Contribution] Adding Lacework Integration

### PR DESCRIPTION
**Pull request by alannix-lw**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This commit adds an integration for the Lacework cloud security platform.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution!
Here are some notes about this PR:
1. For the commands which return a lot of data I would suggest you to add a "limit" argument.
2. If you dont want to add unitests you can delete the Lacework_test.py file.

Let me know if you have any questions

File: Packs/Lacework/Integrations/Lacework/README.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
+Lacework is focused on giving customers visibility and control over their cloud operations at cloud scale to the monitoring of all activities across all cloud components.
+This integration was integrated and tested with version xx of Lacework
```
[the remark this answers is not in the document]
Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/README.md]: Resolved with latest commit.

File: Packs/Lacework/Integrations/Lacework/Lacework.yml [hunk 2]
```diff
@@ -0,0 +1,1098 @@
...
+      description: A string representing the package namespace for which to filter
+        results.
+      isArray: false
+      name: namespace
+      required: false
+      secret: false
+    deprecated: false
+    description: Fetch the host vulnerability information from Lacework.
+    execution: false
+    name: lw-get-host-vulnerabilities
```
[the remark this answers is not in the document]
Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/Lacework.yml:1054]: Resolved with latest commit.

File: Packs/Lacework/Integrations/Lacework/Lacework.yml [hunk 3]
```diff
@@ -0,0 +1,1098 @@
...
+      type: String
+  dockerimage: demisto/lacework:1.0.0.11021
+  feed: false
+  isfetch: true
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
```
[the remark this answers is not in the document]
Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/Lacework.yml:1130]: Resolved with latest commit.

File: Packs/Lacework/Integrations/Lacework/Lacework.py [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+        'EntryContext': ec
+    }
+
+
+def format_compliance_data(compliance_data):
+
+    return CommandResults(
+        outputs_prefix='Lacework.ComplianceReport.recommendations',
+        outputs_key_field=['reportTime'],
+        outputs=compliance_data['data'][0]['recommendations']
```
[the remark this answers is not in the document]
Comment (alannix-lw) [reply Packs/Lacework/Integrations/Lacework/Lacework.py]: Resolved with latest commit.

**Comment by barchen1** [comment]
@alannix-lw I would like to schedule a short demo for us to go over your PR
let me know when is a good time for you where we could have a call.

**Comment by alannix-lw** [comment]
@barchen1 I'm UTC-4, so let me know if there's an agreeable time you think we could meet.  I can also do a screen recording if you let me know what you'd like to see.

**Comment by barchen1** [comment]
> @barchen1 I'm UTC-4, so let me know if there's an agreeable time you think we could meet. I can also do a screen recording if you let me know what you'd like to see.

**Comment by barchen1** [comment]
@alannix-lw Lets talk over Slack and schedule a time for a demo. 
you can also send me an email to bachen@paloaltonetworks.com

**Review by barchen1 — APPROVED** [review]

**Pull request merged**